modules/serverless-deprecated-removed-features.adoc

```diff
@@ -13,21 +13,23 @@ For the most recent list of major functionality deprecated and removed within {S
 // OCP table
 ifdef::openshift-enterprise[]
 .Deprecated and removed features tracker
-[cols="3,1,1,1,1",options="header"]
+[cols="3,1,1,1,1,1",options="header"]
 |====
-|Feature |1.18|1.19|1.20|1.21
+|Feature |1.18|1.19|1.20|1.21|1.22
 
 |`KafkaBinding` API
 |GA
 |Deprecated
 |Deprecated
 |Deprecated
+|Removed
 
 |`kn func emit` (`kn func invoke` in 1.21+)
 |TP
 |TP
 |Deprecated
 |Removed
+|
```

Comment:
I don't think we should leave this blank, let's put removed or a dash or something maybe? I'd check with @nainaz